README.md

```diff
@@ -1 +1,172 @@
+
+### Tech Stack <a name="tech-stack"></a>
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">HTML</a></li>
+  </ul>
+   <ul>
+    <li><a href="#">CSS</a></li>
+  </ul>
+   <ul>
+    <li><a href="#">Javascript</a></li>
+  </ul>
+</details>
+          
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+- **RWD for any devices**
+- **Use Flexbox to place elements in the page**
+- **Use GitHub Pages to deploy web pages**
+- **Use JavaScript to manipulate DOM elements**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+- To install a duplicate of this project on your computer, kindly abide by the guidelines provided below.
+
+### Prerequisites
+
+- To launch the project, please clone my repository onto your PC.
+
+### Setup
+
+ Clone this repository to your desired folder:
+  - Click Code button (shown in green color)
+  - Copy URL.
+  -Easy way(clone it to visual studio code)
+
+### Install
+
+- Install VS code, Node.js and git on your local computer.
+
+### Usage
+
+- In order to run this project, please clone this repository to your PC and open index.html file.
+
+### Run tests
+
+To trace linter errors in VS code, please use the following commands:
+
+- For HTML:
+  
+```
+npx hint .
+```
+
+- For CSS:
+  
+```
+npx stylelint "**/*.{css,scss}"
+```
+
+- For Javascript:
+  
+```
+npx eslint .
+```
+
+### Deployment
+
+- You can deploy this project using: gitHub pages
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+  <!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Hsu Waddy Htat**
+
+- GitHub: [@githubhandle](https://github.com/Hsuwaddyhtat22)
+- Twitter: [@twitterhandle](https://twitter.com/HtatWaddy83921)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/mwlite/profile/in/hsu-waddy-313a40222)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add more webpages such as Program, Join, etc.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](https://github.com/mryeminthu/Capstone-Project/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, follow my Facebook Page and subscribe my You Tube Channel.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- I would like to thank Microverse and my coding partners.
+
```

Comment:
- [ ] The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author. Therefore, you should do it in the README of your project. 👍🏾